### PR TITLE
Update readme re. Immutable.js object formatter

### PR DESCRIPTION
**Pull request by akalin-keybase**

### CODE REVIEW

File: shared/README.md [hunk 1]
```diff
@@ -175,7 +180,7 @@ set `KEYBASE_DEV_TOOL_EXTENSIONS` instead of `KEYBASE_DEV_TOOL_ROOTS`,
 and you'll have to use the version subdirectory:
 
 ```
-set -x KEYBASE_DEV_TOOL_EXTENSIONS "$HOME/Library/Application Support/Google/Chrome/Default/Extensions/fmkadmapgofadopljbjfkapdkoienihi/2.5.0_0"
+set -x KEYBASE_DEV_TOOL_EXTENSIONS "$HOME/Library/Application Support/Google/Chrome/Default/Extensions/fmkadmapgofadopljbjfkapdkoienihi/2.5.2_0,$HOME/Library/Application Support/Google/Chrome/Default/Extensions/hgldghadipiblonfkkicmgcbbijnpeog/1.7_0"
 ```
```
Comment (chrisnojima) [inline shared/README.md]: i'd add a a note that those version numbers change automatically when chrome updates them so they might break at any time

Comment (akalin-keybase) [reply shared/README.md]: There's already a note below, but updated it to say that it can happen at any time.

**Comment by zanderz** [comment]
I hadn't heard of fish shell, do i need it?

File: shared/README.md (hunk 1 above)
Comment (buoyad) [reply shared/README.md]: I thought using `KEYBASE_DEV_TOOL_ROOTS` is more stable for this because it omits the version numbers

**Comment by akalin-keybase** [comment]
@zanderz nah...but if you use bash, it's very similar to set the env vars.

**Comment by akalin-keybase** [comment]
IDK what's up with CI but I"m just gonna merge since it's a readme change

**Pull request merged**